src/services/namingService.ts

```diff
@@ -46,11 +61,11 @@ export class AzureNamingService {
    * @param forbidden Regex for characters to remove from name. Defaults to non-alpha-numerics
    * @param replaceWith String to replace forbidden characters. Defaults to empty string
    */
-  public static getSafeResourceName(config: ServerlessAzureConfig, maxLength: number, resourceConfig?: ResourceConfig, suffix: string = "", includeHash = false) {
+  public static getSafeResourceName(options: AzureNamingServiceOptions, maxLength: number) {
```

Comment:
Any reason `maxLength` isn't in the options?

Reply:
Only since the regular get resource name func didn't need it, but it makes more sense to add